ggml-cuda.cu

```diff
@@ -84,7 +84,7 @@ typedef struct {
 static_assert(sizeof(block_q8_0) == sizeof(float) + QK8_0, "wrong q8_0 block size/padding");
 
 #define CUDA_DEQUANTIZE_BLOCK_SIZE 256
-#define CUDA_DMMV_BLOCK_SIZE 32 // dmmv = dequantize_mul_mat_vec
+#define GGML_CUDA_DMMV_BLOCK_X 32 // dmmv = dequantize_mul_mat_vec
```

Comment:
Init `GGML_CUDA_DMMV_BLOCK_Y` to 1 by default

```suggestion
#define GGML_CUDA_DMMV_BLOCK_X 32 // dmmv = dequantize_mul_mat_vec
#ifndef GGML_CUDA_DMMV_BLOCK_Y
#define GGML_CUDA_DMMV_BLOCK_Y 1
#endif
```

(not tested)

Reply:
This redefines the macro and overrides the option provided by the user. I added a comment instead.

Reply:
Wait sorry, I did it wrong.

Reply:
This works as intended.